soci/ztoc.go

```diff
@@ -54,20 +54,22 @@ type FileMetadata struct {
 }
 
 type Ztoc struct {
-	Version             string
-	BuildToolIdentifier string
-
-	Metadata []FileMetadata
-
+	Version              string
+	BuildToolIdentifier  string
 	CompressedFileSize   FileSize
```

Comment:
Should change this to `CompressedArchiveSize`. Same with `UncompressedArchiveSize`. 

https://github.com/awslabs/soci-snapshotter/blob/main/soci/fbs/ztoc.fbs#L46

Reply:
Good catch. All set.